broker-daemon/block-order-worker/index.spec.js

```diff
@@ -140,12 +140,15 @@ describe('BlockOrderWorker', () => {
       expect(worker).to.have.property('engines', engines)
     })
 
-    it('creates an index for the orders store', async () => {
-      expect(SublevelIndex).to.have.been.calledOnce()
+    it('creates indices for the orders store by swaphash and orderId', async () => {
```

Comment:
I would separate these tests (although I know the `calledTwice` thing is a bit awkward)